examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2097,200 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("sales")
+	coll1 := db.Collection("air_alliances")
+
+	err := coll.Drop(context.Background())
+	require.NoError(t, err)
+	err1 := coll1.Drop(context.Background())
+	require.NoError(t, err1)
+
+	{
+		// Start Aggregation Example 1
+		pipeline := bson.A{
+			bson.D{{"$match", bson.D{{"items.fruit", "banana"}}}},
+			bson.D{{"$sort", bson.D{{"date", 1}}}},
+		}
+
+		cs, err := coll.Aggregate(ctx, pipeline)
+		require.NoError(t, err)
```

Comment:
Wasn't sure how much should be part of the example... Let me know if the `Aggregation` examples should end before this `require` instead.